dist/Client.js

```diff
@@ -238,10 +264,15 @@ class APIClient {
 
 
   async getRasterById(rasterId) {
+    if (!uuidValidator(rasterId)) {
```

Comment:
I don't quite get the point of validating by UUID here. Anyway, it's going to return a 404.